### PR TITLE
Dungeon Game 4.1 and 5.1

### PR DESCRIPTION
**Pull request by McMuzil**

### CODE REVIEW

File: Yurii_Smazhnyi/3/DungeonGame/main.py [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+    Formatted shortcut for printing game_map.
+
+    :returns: None.
+    :rtype: None.
+    """
+
+    global game_map
+
+    for i in range(len(game_map)):
+        print((game_map[i]))
```
Comment (oleksandr-medviediev) [inline Yurii_Smazhnyi/3/DungeonGame/main.py]: Use logging instead of printing

File: Yurii_Smazhnyi/3/DungeonGame/main.py [hunk 2]
```diff
@@ -0,0 +1,319 @@
+import map_generator
+import dungeon_input
+import custom_log
+import serializer
+
```
Comment (oleksandr-medviediev) [inline Yurii_Smazhnyi/3/DungeonGame/main.py]: 2 blank lines after imports

File: Yurii_Smazhnyi/3/DungeonGame/main.py [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+    :text type: str.
+    :param index: index of character.
+    :index type: int.
+    :param replacement: character to replace.
+    :replacement type: str.
+    :returns: result of check.
+    :rtype: bool.
+    """
+
+    return '%s%s%s'%(text[:index],replacement,text[index+1:])
```
Comment (oleksandr-medviediev) [inline Yurii_Smazhnyi/3/DungeonGame/main.py]: It's so ugly...

File: Yurii_Smazhnyi/3/DungeonGame/main.py [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+    Function where whole game is happening.
+
+    :returns: None.
+    :rtype: None.
+    """
+    global is_treasure_near_player
+    global is_trap_near_player
+    global game_map
+
+    custom_log.logger.info("---------------------------------------------------")
```
Comment (oleksandr-medviediev) [inline Yurii_Smazhnyi/3/DungeonGame/main.py]: Why do you need this?

File: Yurii_Smazhnyi/3/DungeonGame/main.py (hunk 3 above)
Comment (McMuzil) [reply Yurii_Smazhnyi/3/DungeonGame/main.py]: don't know how to make it better.

File: Yurii_Smazhnyi/3/DungeonGame/main.py (hunk 4 above)
Comment (McMuzil) [reply Yurii_Smazhnyi/3/DungeonGame/main.py]: to separate turns in console.

**Pull request merged**